chef/cookbooks/ohai/files/default/plugins/linux/crowbar_libvirt.rb

```diff
@@ -0,0 +1,23 @@
+provides "crowbar_ohai"
+
+require_plugin "kernel"
+require_plugin "dmi"
+require_plugin "linux::s390x"
+
+libvirt_uuid = nil
+
+if kernel[:machine] == "s390x"
+  if s390x[:system][:manufacturer] == "KVM"
+    libvirt_uuid = s390x[:system][:uuid]
+  end
+else
+  manufacturer = dmi[:system][:manufacturer] rescue "unknown"
```

Comment:
manufacturer = dmi[:system] ? dmi[:system][:manufacturer] : "unknown"

In this case I think I agree with Hound for the reasons stated in the link it gives.  Having said that, I just realised that I think we had decided to keep Hound as an optional status check, not a required one.